src/coredump/coredumpctl.c

```diff
@@ -915,13 +960,18 @@ static int save_core(sd_journal *j, FILE *file, char **path, bool *unlink_temp)
         /* Look for a coredump on disk first. */
         r = sd_journal_get_data(j, "COREDUMP_FILENAME", (const void**) &data, &len);
         if (r == 0) {
+                char *resolved = NULL;
```

Comment:
For safety we should add `_cleanup_free_` here. It's not strictly necessary right now but it doesn't hurt either